homeassistant/components/rituals_perfume_genie/__init__.py

```diff
@@ -9,14 +10,16 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
-from .const import ACCOUNT_HASH, DOMAIN
+from .const import ACCOUNT_HASH, COORDINATORS, DEVICES, DOMAIN, HUB, HUBLOT
 
-_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["switch", "sensor"]
 
 EMPTY_CREDENTIALS = ""
 
-PLATFORMS = ["switch"]
+_LOGGER = logging.getLogger(__name__)
+UPDATE_INTERVAL = timedelta(seconds=30)
 
 
 async def async_setup(hass: HomeAssistant, config: dict):
```

Comment:
You can drop `async_setup` since it does nothing

Reply:
I had to do a rebase.